src/Modules/Trinity.DynamicCluster/Trinity.DynamicCluster/ChunkedStorage.cs

```
@@ -57,98 +188,315 @@ public override void Dispose()
 
         public override TrinityErrorCode GetCellType(long cellId, out ushort cellType)
         {
-            throw new NotImplementedException();
+            List<ushort> templist = new List<ushort>();
+
+            foreach (Storage.Storage s in PickStorages(a => Chunks[a].Covers(cellId)))
+            {
+                try
+                {
+                    TrinityErrorCode err = s.GetCellType(cellId, out cellType);
```

oops, err is ignored here. I guess it could be E_CELL_NOT_FOUND?